### PR TITLE
use Product Migration instead of SP Migration

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/14491

# Description

See title.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Partially fixes https://github.com/SUSE/spacewalk/issues/14491

### CODE REVIEW

File: modules/client-configuration/pages/client-upgrades-sp-migration.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[client-upgrades-spmigration]]
-= Service Pack Migration
```
Comment (parlt91) [inline modules/client-configuration/pages/client-upgrades-sp-migration.adoc:2]: If it does not mean to much trouble. Can we also rename the file to something like `client-upgrated-product-migration`

File: modules/reference/pages/systems/system-details/sd-sp-migration.adoc [hunk 2]
```diff
@@ -1,7 +1,8 @@
 [[ref-systems-sd-migration]]
-= SP Migration
```
Comment (parlt91) [inline modules/reference/pages/systems/system-details/sd-sp-migration.adoc:2]: Same remark about the file name as above

**Review by parlt91 — COMMENTED** [review]
Thank you for taking care of this :smiley: 
I left some remarks below

File: modules/reference/pages/systems/system-details/sd-sp-migration.adoc (hunk 2 above)
Comment (keichwa) [reply modules/reference/pages/systems/system-details/sd-sp-migration.adoc:2]: yes, I'll do it.   I'll commit it separately.

**Pull request merged**